### DISCUSSION TITLE
0.3.3 maintenance release

### DISCUSSION THREAD

**Discussion by Sturmflut** [lensfun/lensfun#1653] [General]
Pull request #1651 has a release candidate for a new 0.3.3 maintenance release in the 0.3.x branch. It doesn't have any new code, just an updated database and some minor fixes.

I've been playing a bit on https://github.com/Sturmflut/lensfun/releases/tag/v0.3.3-RC1 to test the Github Releases feature. Turns out it is really simply, you don't even have to package the source code.

**Comment by Sturmflut** [comment]
...and we have the first stable release in 6 years, 3 months and 7 days :)

**Reply by gaaned92** [reply]
hi
I am an ART and RawTherapee user. I also produce W10 nightly builds for those two raw processors.

 I regularly build  locally the last commit of master branch of lensfun which is now v0.3.95-863 commit hash 8d12bf8a. 

I was wondering what prevents the Lensfun team to tag a new stable release on the master branch? For instance, a 0.4.0 tag.
Doing that, you will permit to linux packagers to build an up to date lensfun version and not a 0.3.3 which is about 7 years old and is detrimental to Linux users.

**Comment by daneov** [comment]
Wonderful, @Sturmflut! Some time has passed, seems like a lot of work has been done between Feb and now again.

Is there a certain milestone that needs to be achieved before a new release can get tagged? Anxiously awaiting all those new profiles :)

**Reply by Sturmflut** [reply]
The next release is being worked on in https://github.com/lensfun/lensfun/milestone/2 . But you can update your local profile database using the `lensfun-update-data` command at any time.
